src/scripts/main.js

```diff
@@ -1,3 +1,46 @@
 'use strict';
 
 // write code here
+
+const list = document.querySelectorAll('li');
+const container = document.querySelector('ul');
+
+container.innerHTML = '';
+
+const emplList = getEmployees(sortList(list));
+
+for (const i of emplList) {
+  const li = document.createElement('li');
+
+  li.setAttribute('data-position', i['data-position']);
+  li.setAttribute('data-salary', i['data-salary']);
+  li.setAttribute('data-age', i['data-age']);
+  li.innerText = i.name;
+  container.appendChild(li);
+}
+
+function sortList(ls) {
+  const sortedLst = [...ls].sort((a, b) => {
+    const fS = parseFloat(a.getAttribute('data-salary').replace(/[$,]/g, ''));
+    const sS = parseFloat(b.getAttribute('data-salary').replace(/[$,]/g, ''));
+
+    return sS - fS;
+  });
+
+  return sortedLst;
+}
```

Comment:
The `sortList` function sorts the list items correctly, but it should also append the sorted items back to the DOM as required by the task. Consider modifying this function to update the DOM with the sorted list.